pkg/utils/utils.go

```diff
@@ -26,15 +26,22 @@ import (
 )
 
 func RandDate(rnd *rand.Rand) string {
-	return RandTime(rnd).Format("2006-01-02")
+	min := time.Date(1970, 0, 0, 0, 0, 0, 0, time.UTC).Unix()
```

Comment:
Please move it's initiation outside of the function

Reply:
Done